lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -733,8 +733,8 @@ def define
             end
           end
 
-          if ceremony_country == 'italy' && resident_of != 'uk'
-            phrases << :required_supporting_documents_italy
+          if resident_of != 'uk' && data_query.phrase_exists?("required_supporting_documents_#{ceremony_country}")
```

Comment:
I'm testing a new approach here. Instead of relying on a hardcoded list of countries that have exceptions and trusting those phrases are in the yml file, this relies purely on the exceptional phrase being present in the yml file.

What do you think about this approach?